.github/workflows/release.yml

```diff
@@ -0,0 +1,56 @@
+name: 'Release Driver'
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  actions: write
+  contents: write
+  deployments: write
+  packages: write
+  pull-requests: write
+  repository-projects: write
+
+jobs:
+  ubuntu-latest-aurora-build-and-release:
+    name: 'Build And Release Driver'
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Clone Repository'
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 50
+      - name: 'Set up JDK 8'
+        uses: actions/setup-java@v1
+        with:
+          java-version: 8
+      - name: 'Build Driver'
+        run: |
+          ./gradlew --no-parallel --no-daemon -x test build
+      - name: 'Set Version Env Variable'
+        run: echo "RELEASE_VERSION=${GITHUB_REF#refs/*/}" >> $GITHUB_ENV
+      - name: 'Get Release Details'
+        run: |
+          export RELEASE_DETAILS="$(awk -vN=2 'n<N;/^## /{++n}' CHANGES)"
+          export RELEASE_DETAILS="$(sed '${/^# /d;}' <<< "$RELEASE_DETAILS")"
+          export RELEASE_DETAILS="$(sed '$d' <<< "$RELEASE_DETAILS")"
+          touch RELEASE_DETAILS.md
+          echo "$RELEASE_DETAILS" > RELEASE_DETAILS.md
+      - name: 'Install GPG Secret Key'
+        id: install-secret-key
+        run: |
+          cat <(echo -e "${{ secrets.GPG_SECRET_KEY }}") | gpg --batch --import
+      - name: 'Sign Jars'
+        run: |
+          export GPG_TTY=$(tty)
```

Comment:
Do we still need this if the GPG key doesn't require a password?

Reply:
Yes, it's used for signing.